README.md

```diff
@@ -1,4 +1,19 @@
-# Ashley, a self-hosted alternative discussion forum for OpenEdx
+# Ashley, a self-hosted alternative discussion forum for learning
+
+## Overview
+
+`Ashley` is a forum designed to be integrated into `Learning Management Systems` (LMS).
+
+It is a multi-tenant service intended to be operated independently. It can be integrated to any LMS (like [Open edX](https://open.edx.org) or [Moodle](https://moodle.org)), thanks to the [LTI](https://en.wikipedia.org/wiki/Learning_Tools_Interoperability) standard for interoperability.
```

Comment:
Interoperability is already a keyword from the LTI acronym.

```suggestion
It is a multi-tenant service intended to be operated independently. It can be integrated to any LMS (like [Open edX](https://open.edx.org) or [Moodle](https://moodle.org)), thanks to the [LTI](https://en.wikipedia.org/wiki/Learning_Tools_Interoperability) standard.
```